en/services/gimbal_v2.md

```diff
@@ -235,6 +247,14 @@ For RC control, the channels will have to be manually mapped/configured to contr
 This is the same approach as is used for managing the input source for flying; it is up to the implementation to select either RC or MAVLink.
 The recommendation is to make it configurable using (for instance) a parameter.
 
+#### What about non-MAVLink gimbals
+
+A non-MAVLink gimbal needs to be connected to a gimbal manager which then takes care of sending the gimbal device messages.
+Since a non-MAVLink gimbal can't be addressed with a MAVLink component ID, the `gimbal_device_id` needs to be set to the magic value 1 to 6 which signals that the gimbal manager also "is" (or fakes to be) the gimbal device.
+
+Also see [how to address non-MAVLink gimbal devices](#non_mavlink_gimbal_device_addressing).
```

Comment:
```suggestion
#### Non-MAVLink gimbals

A non-MAVLink gimbal needs to be connected to a gimbal manager, which then takes care of sending the gimbal device messages.
Since a non-MAVLink gimbal can't be addressed with a MAVLink component ID, the `gimbal_device_id` needs to be set to one of the magic values (1 to 6).
This signals that the gimbal manager also is also acting as the gimbal device.

Also see [how to address non-MAVLink gimbal devices](#non_mavlink_gimbal_device_addressing).
```